.github/ISSUE_TEMPLATE_tmpl.md

```diff
@@ -28,9 +28,12 @@
 
 ### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:
 
-Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
+Add `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
+
+<details>
+<summary>Log</summary>
```

Comment:
Verbose output is the central part of the report and it should not be collapsed by default. As a developer I don't want to take any additional actions to make it visible.